autoload/tagbar.vim

```diff
@@ -879,14 +879,21 @@ function! s:OpenWindow(flags) abort
     endif
 
     let s:window_opening = 1
-    if g:tagbar_vertical == 0
+    if g:tagbar_position =~# '\v(bottom|right)'
+        let openpos = 'rightbelow '
+    else
+        let openpos = 'leftabove '
+    endif
+    if g:tagbar_position =~# '\v(left|right)'
         let mode = 'vertical '
```

Comment:
I understand this isn't part of the code you added, but something doesn't sit right with me here. Why is this defined with a space at the end of the variable's contents? Why is the other option '' (instoad of 'horizontal')?

This PR looks great over-all and is an improvement, but I'm wondering if perhaps there is a little bit more that could be cleaned up at the same time (separate commit maybe, but same PR). Cat we make even these local variables sane while we're mucking with this?

Reply:
My intent was to leave the commands being executed the same as much as possible. The extra spaces at the end are because they are all being concatenated together below at the `exe 'silent keepalt ' . openpos . mode . width . 'split ' . sTagbarBufName()` line. If you have any suggestions for how you'd like this cleaned up, I can certainly take a shot at it.

Reply:
Also in reference to other option being '' instead of 'horizontal', that is because of the `split` command being executed. In vim, there is no `split horizontal` command. It is either `split` or `split vertical`. This code is constructing a command to be executed in vim to split the window according to the requested parameters.

Reply:
Thanks for the feedback on this. I see that's kind of a hacky way to make this happen but understand why it is the way it is. I think for now you might be right, the best thing is to just leave it alone.